functions/tests/testdata/key_values.flux

```diff
@@ -0,0 +1,4 @@
+t_key_values = (table=<-) =>
+  table
+  |> keyValues(keyColumns: ["_value"])
```

Comment:
The canonical use of this function is to replace  "SHOW TAG VALUES" in `influxql`.   Can we add a second test that runs  `keyValues(keyColumns: ["host", "name"])`

Reply:
No problem. The second test has been added.